.github/workflows/ci.yaml

```diff
@@ -76,6 +76,7 @@ env:
     test_math
     test_operator
     test_ordered_dict
+    test_os
```

Comment:
test_os is platform dependent.

```suggestion
```

Reply:
But looks like the test pass for other platforms in CI as well.
https://github.com/derlih/RustPython/blob/0e68f9763f862ec3fdff5f6adb6837eeec76bd00/.github/workflows/ci.yaml#L286-L299

Reply:
Oh, yep. Now I got what you intended.

"platform independent" here means it always generate same result regardless of the platform. In that case, we only run the tests on Ubuntu to keep up our CI capacity.
About test_os, even if every tests passes on every platforms, it is still platform dependent, because we need to run the test in every os to ensure tests are not broken from any OS.

Maybe we'd better to add a comment around the list not to make confusion in future.

Reply:
I suggest renaming this variable to `ONLY_TEST_THESE_ON_LINUX`.

Reply:
The name doesn't describe why we run then only on linux. Either way we need description.

Reply:
Then how about `THESE_TESTS_ARE_NOT_EXPECTED_TO_CHANGE_BEHAVIOR_REGARDLESS_OF_WHATEVER_OS_THEY_RUN_ON`?

Reply:
There are other modules that involve some platform code like `socket`, `io`, e.t.c. May be it would be better to create another env variable `OS_TESTS` or `PLATFORM_TESTS`?

Reply:
@youknowone @fanninpm 
What should be done with this change?

Reply:
Removing `test_os` is enough in this PR. We will handle other stuffs later.